default/python/universe_generation/galaxy.py

```diff
@@ -45,32 +307,10 @@ def calc_universe_width(shape, size):
     return width
 
 
-def get_systems_within_jumps(origin_system, jumps):
-    """
-    Returns all systems within jumps jumps of system origin_system (including origin_system).
-    If jumps is 0, return list that only contains system origin_system.
-    If jumps is negative, return empty list.
-    """
-    if jumps < 0:
-        return []
-    matching_systems = [origin_system]
-    next_origin_systems = [origin_system]
-    while jumps > 0:
-        origin_systems = list(next_origin_systems)
-        next_origin_systems = []
-        for system in origin_systems:
-            neighbor_systems = [s for s in fo.sys_get_starlanes(system) if s not in matching_systems]
-            next_origin_systems.extend(neighbor_systems)
-            matching_systems.extend(neighbor_systems)
-        jumps -= 1
-    return matching_systems
```

Comment:
Ok, now I get where you're coming from. And considering that `galaxy.py` only contains star system placement code besides `get_systems_within_jumps` your conclusion is understandable, however, the `galaxy.py` module is intended to contain *all* the galaxy map related stuff, hence the name. Which coincidentally is just star system placement and the lonely `get_systems_within_jumps` function.

When I wrote that code I was actually aware of the slight ackwardness of that combination, and considered splitting up things differently. But `get_systems_within_jumps` doesn't belong into `empires.py`, this module contains empire setup stuff, not galaxy map stuff. And creating a module just for that single function seemed overkill.

So I ended up putting these together into a module that's supposed to contain the map related stuff. I'm open to suggestions for a better name though, if you think `galaxy.py` is misleading.